std/string.d

```diff
@@ -3171,108 +3171,269 @@ S detab(S)(S s, size_t tabSize = 8) @trusted pure
     Params:
         s       = String to convert.
         tabSize = Tab columns are $(D tabSize) spaces apart.
+
+    Returns:
+        GC allocated string with tabs replaced with spaces;
+        use $(LREF entabber) to not allocate.
+
+    See_Also:
+        $(LREF entabber)
  +/
-S entab(S)(S s, size_t tabSize = 8) @trusted pure
+S entab(S)(S s, size_t tabSize = 8) @trusted
     if (isSomeString!S)
 {
-    import std.utf : encode;
-    import std.uni : lineSep, paraSep;
-    import std.exception : assumeUnique;
+    import std.array;
+    return cast(S)(entabber(s, tabSize).array);
+}
 
-    bool changes = false;
-    alias C = Unqual!(typeof(s[0]));
-    C[] result;
+///
+unittest
```

Comment:
That's purely a documentation unittest. The real ones are later and are so marked.